server-2041 | updated readme (#163)
CircleCI-Public/server-terraform: nomad-aws/README.md

  # enable_irsa input will allow K8s service account to use IAM roles, you have to replace REGION, ACCOUNT_ID, OIDC_ID and K8S_NAMESPACE with appropriate value
  # for more info, visit - https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html
  enable_irsa = {}

>   ssh_key = "<< public key which will attched to each nomad client >>"

Reviewer: ```suggestion
  ssh_key = "<< public key to be placed on each nomad client >>"
```
